### ISSUE TITLE
[Raid - ICC25 HC][NPC][Spell] Soul Reaper

### ISSUE THREAD

**Issue by ih8mondays** [dalaranwow/dalaran-wow#5831]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**:
The Soul Reaper tick does about 20k.

**Expected behaviour**:
Sould Reaper should hit for 50% weapon damage from Lich King and tick for 70,000 shadow damage after 5 seconds (25m Heroic)
Source: https://wowwiki.fandom.com/wiki/Lich_King_(Icecrown_Citadel_tactics)

**Steps to reproduce the problem**:
1. Pull LK 25 HC.
2. Get hit by Soul Reaper during phase 2 or 3.
3. Check the damage it does and remember to account for reductions.

**Include proofs for this behaviour**
World of Logs for Soul Reaper 17/11-2019: https://imgur.com/a/0bDkV7y
The log goes: LK casts -> 50% hit -> Aura Applied -> Tick -> Aura Removed.   Rinse and repeat.

The initial hit seems a little low and should probably also be looked at, but I'm focusing on the tick.
As you can see the tick only does just below 20k including absorbs and resists instead of the 70k it is supposed to.

[//]: # (You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/)
[//]: # (Issues without proofs will be closed and tag On Hold will be added until the proof is provided.)

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
https://wowgaming.altervista.org/aowow/?spell=73799
https://wotlk-twinhead.twinstar.cz/?spell=73799

[//]: # (You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/)
[//]: # (**Include Screenshots from the issue if necessary**)

**Comment by Lothloryen** [comment]
Hello there,

Thank you for the report, I will take a look into this.

**Comment by wootlekins** [comment]
Any updates on this?

**Comment by Lothloryen** [comment]
Hello there,

The issue should be fixed after the next scheduled mainteance.

**Comment by mrnuggets** [comment]
> 
> 
> Hello there,
> 
> The issue should be fixed after the next scheduled mainteance.

I'd suggest going over this again, Soul Reaper is only hitting for 20-25k, and soul reaper should also strike for 50% weapon damage once applied, which it doesn't.

https://wotlk.evowow.com/?spell=73799 read the spell.
